README.md

```diff
@@ -1,3 +1,29 @@
+# sync-endpoint-default-setup
+
+## What is sync-endpoint-default-setup
+sync-endpoint-default-setup is a project that provides a default setup and configuration for the ODK-X Sync Endpoint. ODK-X is a suite of tools and services for data collection, management, and synchronization, often used for field data collection in research, humanitarian work, and other applications.
```

Comment:
@wbrunette is the language here correct? Would we refer to this as a project?. TOC otherwise looks good.

Reply:
Yes we would refer to "sync-endpoint-default-setup" as a project.